Automattic/jetpack
# JITM: Update styles

Update the JITM design.

#### Changes proposed in this Pull Request:

* Update the JITM styles.

#### Other information:


#### Jetpack product discussion

p6TEKc-6F7-p2

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test locally
* Go to any page that shows JITMs, like the `Plugins` page or the Jetpack Dashboard
* Wait some seconds for the JITMs to be shown
* Check that the styles are updated according to the Figma specs

MrkHtDbIWxPThpI1BbaRhH-fi-2821%3A7030

Before

![2023-01-26_16-14-16](https://user-images.githubusercontent.com/8486249/214928414-bcc412ff-7c15-422f-a205-ab9cce1d3761.png)

After

![2023-01-26_16-03-14](https://user-images.githubusercontent.com/8486249/214928695-f9b172b3-5f9b-469a-9e7c-634c74cd7d46.png)
![2023-01-26_16-02-51](https://user-images.githubusercontent.com/8486249/214928703-364c2b0b-ee2f-4e8a-afe0-1f11522722bc.png)
![2023-01-26_15-12-49](https://user-images.githubusercontent.com/8486249/214928711-d4cd5fd7-18c4-4c67-87bb-048aac70a778.png)
![2023-01-26_15-11-03](https://user-images.githubusercontent.com/8486249/214928717-a7ba170e-c0d8-4d82-904a-1ce623701841.png)

Figma

![2023-01-26_16-12-39](https://user-images.githubusercontent.com/8486249/214928811-91e12256-123a-449e-b165-41a96e81302b.png)
